e2e host FUSE tests failing on Fedora 32

### Version of Singularity:

What version of Singularity are you using? Run:

```
3.6.0-rc.4+13-g31aac78a8
```

### Actual behavior

```

            --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount (9.62s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostDaemonAsRoot (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostAsRoot (0.09s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerDaemonAsRoot (0.17s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerAsRoot (0.15s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostDaemonAsUser (0.07s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostAsUser (0.07s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerDaemonAsUser (0.15s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerAsUser (0.14s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostDaemonAsUserNamespace (0.08s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostAsUserNamespace (0.09s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerDaemonAsUserNamespace (0.16s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerAsUserNamespace (0.15s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostDaemonAsFakeroot (0.09s)
                --- FAIL: TestE2E/PAR/ACTIONS/fuse_mount/HostAsFakeroot (0.08s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerDaemonAsFakeroot (0.15s)
                --- PASS: TestE2E/PAR/ACTIONS/fuse_mount/ContainerAsFakeroot (0.15s)
```

```
/usr/local/bin/singularity exec --fusemount "host:/tmp/stest.170437611/sshfs-139078343/sshfs-wrapper root@127.0.0.1:/ -p 2022 -o IdentityFile=/tmp/stest.170437611/sshfs-139078343/etc/ssh/ssh_host_rsa_key -o StrictHostKeyChecking=no /mnt" /tmp/stest.170437611/sshfs-139078343 test -d /mnt/etc
/etc/crypto-policies/back-ends/openssh.config: line 3: Bad configuration option: gssapikexalgorithms
/etc/crypto-policies/back-ends/openssh.config line 5: Bad key types 'rsa-sha2-256,rsa-sha2-256-cert-v01@openssh.com,ecdsa-sha2-nistp256,ecdsa-sha2-nistp256-cert-v01@openssh.com,sk-ecdsa-sha2-nistp256@openssh.com,sk-ecdsa-sha2-nistp256-cert-v01@openssh.com,ecdsa-sha2-nistp384,ecdsa-sha2-nistp384-cert-v01@openssh.com,rsa-sha2-512,rsa-sha2-512-cert-v01@openssh.com,ecdsa-sha2-nistp521,ecdsa-sha2-nistp521-cert-v01@openssh.com,ssh-ed25519,ssh-ed25519-cert-v01@openssh.com,sk-ssh-ed25519@openssh.com,sk-ssh-ed25519-cert-v01@openssh.com,ssh-rsa,ssh-rsa-cert-v01@openssh.com'.
read: Connection reset by peer
lstat //mnt: transport endpoint is not connected
```

--- a/e2e/actions/actions.go
+++ b/e2e/actions/actions.go
@@ -1714,7 +1714,8 @@ func (c actionTests) fuseMount(t *testing.T) {
 		},
 	}
 
-	optionFmt := "%s:%s root@127.0.0.1:/ -p 2022 -o IdentityFile=%s -o StrictHostKeyChecking=no %s"
+	optionFmt := "%s:%s root@127.0.0.1:/ -p 2022 -F %s -o IdentityFile=%s -o StrictHostKeyChecking=no %s"
+	sshConfig := filepath.Join(imageDir, "etc", "ssh", "ssh_config")
 
 	for _, tt := range basicTests {
 		c.env.RunSingularity(
@@ -1723,7 +1724,7 @@ func (c actionTests) fuseMount(t *testing.T) {
 			e2e.WithProfile(tt.profile),
 			e2e.WithCommand("exec"),
 			e2e.WithArgs([]string{
-				"--fusemount", fmt.Sprintf(optionFmt, tt.spec, sshfsWrapper, tt.key, "/mnt"),
+				"--fusemount", fmt.Sprintf(optionFmt, tt.spec, sshfsWrapper, sshConfig, tt.key, "/mnt"),
 				imageDir,
 				"test", "-d", "/mnt/etc",
 			}...),

